--- project-spec/configs/rootfs_config	2025-03-10 02:20:39.334348918 -0400
+++ project-spec/configs/rootfs_config2	2025-03-10 02:55:22.918288484 -0400
@@ -15,7 +15,7 @@
 #
 # sudo 
 #
-# CONFIG_sudo is not set
+CONFIG_sudo=y
 # CONFIG_sudo-dbg is not set
 # CONFIG_sudo-dev is not set
 
@@ -194,7 +194,7 @@
 #
 # kmod 
 #
-# CONFIG_kmod is not set
+CONFIG_kmod=y
 # CONFIG_kmod-dbg is not set
 # CONFIG_libkmod is not set
 # CONFIG_kmod-dev is not set
@@ -490,7 +490,7 @@
 #
 # lrzsz 
 #
-# CONFIG_lrzsz is not set
+CONFIG_lrzsz=y
 # CONFIG_lrzsz-dbg is not set
 # CONFIG_lrzsz-dev is not set
 
@@ -3676,7 +3676,6 @@
 # CONFIG_matchbox-theme-sato is not set
 # CONFIG_matchbox-theme-sato-dev is not set
 # CONFIG_matchbox-theme-sato-dbg is not set
-
 #
 # linuxptp 
@@ -490,7 +490,7 @@
 #
 # lrzsz 
 #
-# CONFIG_lrzsz is not set
+CONFIG_lrzsz=y
 # CONFIG_lrzsz-dbg is not set
 # CONFIG_lrzsz-dev is not set
 
@@ -3705,7 +3705,7 @@
 #
@@ -3695,6 +3694,9 @@
 # Petalinux Package Groups
 #
 
+CONFIG_packagegroup-petalinux=y
+CONFIG_packagegroup-petalinux-utils=y
+
 #
 # packagegroup-xilinx-audio 
 #
@@ -3775,7 +3777,10 @@
 #
 # PetaLinux RootFS Settings
 #
-CONFIG_ADD_EXTRA_USERS="root:root;petalinux::passwd-expire;"
+# CONFIG_ADD_EXTRA_USERS="root:root;petalinux::passwd-expire;"
+CONFIG_ADD_EXTRA_USERS="root:root;shim:shim;"
 CONFIG_CREATE_NEW_GROUPS="aie;"
-CONFIG_ADD_USERS_TO_GROUPS="petalinux:audio,video,aie,input;"
-CONFIG_ADD_USERS_TO_SUDOERS="petalinux"
+# CONFIG_ADD_USERS_TO_GROUPS="petalinux:audio,video,aie,input;"
+# CONFIG_ADD_USERS_TO_SUDOERS="petalinux"
+CONFIG_ADD_USERS_TO_GROUPS="shim:audio,video,aie,input;"
+CONFIG_ADD_USERS_TO_SUDOERS="shim"
